ts/screens/profile/RemoveAccountInfoScreen.tsx

```diff
@@ -46,5 +46,7 @@ const RemoveAccountInfo: React.FunctionComponent<Props> = props => {
     </BaseScreenComponent>
   );
 };
-
-export default RemoveAccountInfo;
+const mapDispatchToProps = (dispatch: Dispatch) => ({
+  loadBonusAndProceed: () => dispatch(loadBonusBeforeRemoveAccount())
```

Comment:
What happen if the API response with a delay? we press the button but nothing happens
To take it easy I suggest to remove the navigation from saga `handleLoadBonusBeforeRemoveAccount` and place it there.
So you at the same time

- ask for bonus data
- navigate to the next screen

If we can't load data the alert won't be shown. But I guess it is acceptable